docs/components/modeler/forms/configuration/forms-config-templating-syntax.md

```diff
@@ -0,0 +1,136 @@
+---
+id: forms-config-templating-syntax
+title: Templating syntax
+description: Learn about templated properties configuration, which provides dynamic content creation within forms using a templating language called feelers.
+---
+
+Templated properties configuration allows for dynamic content creation within forms using a templating language called **feelers**.
```

Comment:
Let's link the `feelers` project here, as readers might be interested in the details.